src/_DataStructures_/LinkedList/index.js

```diff
@@ -13,7 +13,7 @@ class LinkedList {
   }
 
   addAtBeginning(element) {
-    this.head = new Node(element, this.head);
+    this.head=new Node(element, this.head);
```

Comment:
Can you undo these changes because eslint will show this error `Operator '=' must be spaced.eslint(space-infix-ops)`